src/main/perl/TextRender.pm

```diff
@@ -154,16 +153,10 @@ Java properties format (using C<Config::Properties>),
 
 .INI format (using C<Config::Tiny>)
 
-=item general
-
-(using C<Config::General>).
-
-Caution: there is no way to garantee reproducible results as
-there is no control over the order of hash keys. It is recommended to
-try to generate this format in another way (e.g. via TT).
-
 =back
 
+(Builtin module <general> is provided as part of L<CCM::TextRender> since 15.10).
```

Comment:
Is the old functionality available via the CCM::TextRender module? While CAF::* is a public API, I would have thought CCM::TextRender is a private API for manipulating CCM files so we shouldn't imply that it's useable. In which case I would just stop at: "... was removed in 15.12, see http://github.com/issue/... for details."

Or is it intended that component writers can call upon the functionality available in CCM::TextRender if they wish? In which case I would say: "... was removed in 15.12. Component writers needing this functionality can use the L<CCM::TextRender> subclass instead." (But that seems strange - why did we ditch it if it's still available?)